src/LabelSelector/LabelChip/LabelChip.tsx

```diff
@@ -12,31 +13,36 @@ export default function LabelChip({
   size = "medium",
   variant = "filled",
   visible = true,
+  selected = false,
+  description = "",
   ...props
 }: LabelChipProps) {
   // return the styled chip component
   return (
-    <Chip
-      {...props}
-      className="label-chip"
-      clickable={clickable}
-      sx={{
-        "&:hover": {
-          backgroundColor: clickable ? darken(color, 0.2) : color
-        },
-        backgroundColor: color,
-        color: theme => theme.palette.getContrastText(color),
-        [`& .${chipClasses.deleteIcon}`]: {
-          color: theme => alpha(theme.palette.getContrastText(color), 0.7)
-        },
-        [`& .${chipClasses.deleteIcon}:hover`]: {
-          color: theme => theme.palette.getContrastText(color)
-        },
-        visibility: visible ? "visible" : "hidden"
-      }}
-      label={<NoWrapTypography variant="inherit">{label}</NoWrapTypography>}
-      size={size}
-      variant={variant}
-    />
+    <Tooltip title={description}>
```

Comment:
The font size for the tooltip in Figma is also different, although we are using the MUI default here which matches tooltips across the rest of our components. What do you want here @Sowbhagya-ipg?

Reply:
I have used the MUI Default too. I wonder why it is not the same. The tooltip font size that I have is 10px.

Reply:
Looks like we updated the default to be 14px 4+ years ago across all tooltips:

https://github.com/IPG-Automotive-UK/react-ui/blob/a742c9a1635c05fa7a631126dbfad76be1479be6/src/ThemeProvider/ThemeProvider.tsx#L69-L76

Reply:
Will wait @Sowbhagya-ipg to comment final decision on this.

Reply:
The tooltip with a 10px is too small and 14px is relatively larger. I would suggest it to to be 12px which is lesser than the body font size we have and also readable. @m4manjesh @mattcorner and font weight